python/cudf/cudf/core/index.py

```diff
@@ -2970,7 +2970,9 @@ def total_seconds(self) -> cupy.ndarray:
 
         This method is currently not implemented.
         """
-        return self._column.total_seconds().values
+        return cudf.Index._from_column(
```

Comment:
Could you change the return type annotation to `-> cudf.Index`?

Reply:
Fixed it 👍